src/companies/tests/conftest.py

```diff
@@ -87,3 +89,30 @@ def reader_client(request) -> ApiClient:
 )
 def as_point_managing_staff(request) -> ApiClient:
     return request.param
+
+
+@pytest.fixture
+def user_fixtures_collection(
+    as_another_company_owner: ApiClient, as_user: ApiClient, as_anon: ApiClient
+) -> dict[str, ApiClient]:
+    return {
+        "as_another_company_owner": as_another_company_owner,
+        "as_user": as_user,
+        "as_anon": as_anon,
+    }
+
+
+@pytest.fixture(
+    params=[
+        ("as_another_company_owner", status.HTTP_403_FORBIDDEN),
+        ("as_user", status.HTTP_403_FORBIDDEN),
+        ("as_anon", status.HTTP_401_UNAUTHORIZED),
+    ]
+)
+def as_point_non_managing_staff(
+    request: FixtureRequest, user_fixtures_collection: dict[str, ApiClient]
+) -> StatusApiClient:
+    user, status = request.param
+    client: ApiClient = user_fixtures_collection[user]
+    client.expected_status = status  # type: ignore[attr-defined]
+    return client  # type: ignore[return-value]
```

Comment:
This will allow us to remove mypy ignore statements
```suggestion
    user, status = request.param
    client: ApiClient = user_fixtures_collection[user]
    statused_client = StatusedClient(status, getattr(client, "user", None)
    return statused_client
```

Reply:
Then we raise in the tests: `sqlite3.IntegrityError: UNIQUE constraint failed: authtoken_token.user_id`. I will try to do a new token for StatusApiClient.